Sources/LanguageServerProtocol/WorkspaceEdit.swift

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public struct WorkspaceEdit: Codable {
```

Comment:
How is this related to workspaceFolders?

Reply:
it's not. Just a model though. I can separate it to the PR later

Reply:
Sounds good